backend/db/session.js

```diff
@@ -31,6 +31,13 @@ const setLocalUserData = (req, res, next) => {
     res.locals.errors = null;
   }
 
+  if (req.session.messages) {
+    res.locals.messages = req.session.messages;
+    delete req.session.messages;
+  } else {
+    res.locals.messages = null;
+  }
```

Comment:
not actually sure what the plan is for passing new messages to the chat. Used sessions for the time being. @ahuda620 @CaiminRO Definitely let me know if this fits into what is needed for the chat.